src/migrations/004.sql

```diff
@@ -0,0 +1,20 @@
+CREATE UNIQUE INDEX ON team_player (player_id);
+
+ALTER TABLE player
+  DROP COLUMN is_vouched;
+
+CREATE TABLE vouch (
+  vouched_id varchar(50) NOT NULL REFERENCES steam_user (steam_id),
+  voucher_id varchar(50) NOT NULL REFERENCES steam_user (steam_id),
+  CHECK(vouched_id != voucher_id)
+);
+
+CREATE TYPE yesnomaybe AS enum('yes', 'no', 'maybe');
```

Comment:
I've mostly used mysql, but the general complaint there was that adding a new value to an enum was very painful and if you ever wanted to add any metadata to the enum (which pretty often happened) you're SoL.  Apparently in postgres 9.1+, you can append to an enum with just an alter table though.